css/modals/video-quality/_video-quality.scss

```diff
@@ -126,6 +126,7 @@
 
     i {
         cursor: pointer;
+        line-height: $videoStateIndicatorSize;
```

Comment:
I was thinking...maybe you should get rid of that cursor? I probably should have killed it with whatever cleanup was being done. It's fine if you don't because it's not related to dark-theme.

Reply:
Yeah, you're right, I will remove it.